src/components/Overview/OverviewEducLink.tsx

```diff
@@ -0,0 +1,85 @@
+// img
+import DownloadDataPic from '../../assets/img/jeremy-bezanger-Ro38zH1CRUY-unsplash.jpg';
+import DataBrokersPic from '../../assets/img/annie-spratt-t859lVr8KY0-unsplash.jpg';
+import InferredDataPic from '../../assets/img/thisisengineering-raeng-8hgmG03spF4-unsplash-520x245.jpg';
+import RiskLocDataPic from '../../assets/img/donald-giannatti-Wj1D-qiOseE-unsplash.jpg';
+import DeleteDataPic from '../../assets/img/sara-kurfess-6lcT2kRPvnI-unsplash.jpg';
+
+//utils
+import styled from 'styled-components';
+
+// Contains each link infomations
+const educLinks = [
```

Comment:
cet array pourrait être dans un fichier de constantes; et il pourrait être typé aussi :) 
```suggestion
interface EducationLink {
  img: Image,
  title: string,
  link: string
}
const educLinks: EducationLink[] = [
```

Reply:
Ok, j'ai attribué une string au lieu d'Image qui me renvoie un message d'erreur (la string fonctionne, car img correspond à une URL selon ce que j'ai lu).
Pour le fichier de constante, il s'agit d'un dossier séparer qui pourrait se localiser dans le dossier utils par exemple ?